web/server/codechecker_server/api/report_server.py

```diff
@@ -2441,6 +2520,7 @@ def checker_is_unavailable(checker_name):
                 detection_status = 'new'
                 detected_at = run_history_time
 
+                old_status = None
```

Comment:
Do we need to set the default value here? The variable is used only inside the if block below I did not found any other references.